### ISSUE TITLE
Quick redesign of simple search

### ISSUE THREAD

**Issue by Ptrhnk** [DISSINET/InkVisitor#1064]

**Comment by Ptrhnk** [comment]
@davidzbiral not sure if this is still needed for this version. I was considering making more space for search results but it seems big enough on external monitor and there's still some space on smaller display. There will be different search in next version so what do you think?

**Comment by davidzbiral** [comment]
Yes, it can wait, @Ptrhnk . The crucial thing was the search algorithm, it was not finding things - but this I hope is fixed.
